discord/types/appinfo.py

```diff
@@ -0,0 +1,70 @@
+"""
```

Comment:
Something about the way these types are duplicated here doesn't sit well with me. Is there no way to reduce the duplication here? The types should mirror this one: <https://discord.com/developers/docs/topics/oauth2#application>